sdk/core/azure-core-management/src/test/java/com/azure/core/management/implementation/polling/LROPollerTests.java

```diff
@@ -324,6 +276,107 @@ public String getName() {
         }
     }
 
+    @Test
+    public void lroRetryAfter() {
+        ServerConfigure configure = new ServerConfigure();
+        Duration expectedPollingDuration = Duration.ofSeconds(3);
+        configure.pollingCountTillSuccess = 3;
+        configure.additionalHeaders = new HttpHeaders(new HttpHeader("Retry-After", "1"));  // 1 second
+        WireMockServer lroServer = startServer(configure);
+        lroServer.start();
+
+        try {
+            final ProvisioningStateLroServiceClient client = RestProxy.create(ProvisioningStateLroServiceClient.class,
+                createHttpPipeline(lroServer.port()),
+                SERIALIZER);
+
+            PollerFlux<PollResult<FooWithProvisioningState>, FooWithProvisioningState> lroFlux
+                = PollerFactory.create(SERIALIZER,
+                new HttpPipelineBuilder().build(),
+                FooWithProvisioningState.class,
+                FooWithProvisioningState.class,
+                POLLING_DURATION,
+                newLroInitFunction(client, FooWithProvisioningState.class));
+
+            long nanoTime = System.nanoTime();
+
+            FooWithProvisioningState result = lroFlux
+                .doOnNext(response -> {
+                    System.out.println(String.format("[%s] status %s",
+                        OffsetDateTime.now().toString(), response.getStatus().toString()));
+                }).blockLast()
+                .getFinalResult().block();
+            Assertions.assertNotNull(result);
+
+            Duration pollingDuration = Duration.ofNanos(System.nanoTime() - nanoTime);
+            Assertions.assertTrue(pollingDuration.compareTo(expectedPollingDuration) > 0);
```

Comment:
If we just check polling duration larger than expected. I think we could make `Retry-After` larger than the original one.

Reply:
Yes. In this test, Retry-After is 1sec, default interval is 100ms.

Reply:
Got it. I had thought it was 30s.